sed/core/config.py

```diff
@@ -0,0 +1,222 @@
+"""This module contains a config library for loading yaml/json files into dicts
+"""
+import json
+import os
+import platform
+from importlib.util import find_spec
+from pathlib import Path
+from typing import Union
+
+import yaml
+
+package_dir = os.path.dirname(find_spec("sed").origin)
+
+
+def parse_config(
+    config: Union[dict, str] = None,
+    folder_config: Union[dict, str] = None,
+    user_config: Union[dict, str] = None,
+    system_config: Union[dict, str] = None,
+    default_config: Union[
```

Comment:
I think the difference between the three configs isn't quite clear, at least to me. So I can imagine it'd be likewise for the users.

Reply:
Ill update the docu and provide examples